sdks/python/src/opik/message_processing/messages.py

```diff
@@ -6,7 +6,8 @@
 
 @dataclasses.dataclass
 class BaseMessage:
-    pass
+    def as_dict(self) -> Dict[str, Any]:
+        return dataclasses.asdict(self)
```

Comment:
create a new dict from `self.__dict__`.
`asdict` triggers values copying (it will fail if one of the fields is unpickleable object, e.g. `threading.Lock`).

And let's rename this method to smth like `as_payload_dict()`, `as_dict` makes impression of just creating a dict from the attributes as is, but in reality we need to change the keys sometimes.